sympy/functions/combinatorial/numbers.py

```diff
@@ -2292,3 +2292,126 @@ def ok(x):
         return Integer(sum(1 for i in multiset_derangements(s)))
     return Integer(abs(integrate(exp(-x)*Mul(*[
         laguerre(i, x)**m for i, m in counts.items()]), (x, 0, oo))))
+
+
+#----------------------------------------------------------------------------#
+#                                                                            #
+#                           Padovan numbers                                  #
+#                                                                            #
+#----------------------------------------------------------------------------#
+
+
+class padovan(Function):
+    r"""
+    In number theory, the Padovan sequence is the sequence of integers P(n)
+    defined by the initial values `P_0 = 1`, `P_1 = 1` , `P_2 = 1` and the
+    the two-term recurrence `P_n = P_{n-2} + P_{n-3}`.
+
+    * ``padovan(n)`` gives the `n^{th}` Padovan number, `P_n`
+
+    Examples
+    ========
+
+    >>> from sympy import padovan
+
+    >>> padovan.is_padovan(2)
+    True
+    >>> [padovan(x) for x in range(0,11)]
+    [1, 1, 1, 2, 2, 3, 4, 5, 7, 9, 12]
+    >>> padovan.find_first_n_padovan_numbers(8)
+    [1, 1, 1, 2, 2, 3, 4, 5]
+
+
+    References
+    ==========
+
+    .. [1] https://en.wikipedia.org/wiki/Padovan_sequence
+    .. [2] https://mathworld.wolfram.com/PadovanSequence.html
+
+    """
+
+    @staticmethod
+    def is_padovan(n):
+        try:
+            n = as_int(n)
+        except ValueError:
+            False
```

Comment:
This should probably be `return False`

Reply:
Oops, I'll change this as soon as possible  .